pkg/tsdb/influxdb/flux/executor.go

```diff
@@ -27,7 +27,7 @@ func executeQuery(ctx context.Context, query queryModel, runner queryRunner, max
 		glog.Warn("Flux query failed", "err", err, "query", flux)
 		dr.Error = err
 	} else {
-		dr = readDataFrames(tables, int(float64(query.MaxDataPoints)*1.5), maxSeries)
```

Comment:
I wonder if we should go even bigger?  the real goal is avoiding selecting way too many things maybe 2.5x or even 5x?

Reply:
It's a bit tough, because 10k datapoints sounds like a lot, but that's the max default for flux query models if it's not set via the frontend, so I think we're safe here. And if you get here via the frontend, then you don't really want more because it can't be drawn properly?

Reply:
the real solution is to add a warning/notice to the result when the data is clipped using this maxDataPoints -- but I think this is a good quick fix

Reply:
Ha, there is an error generated when it's clipped, but the error is overwritten AFAI can tell: https://github.com/grafana/grafana/blob/5f1f820bb99eeca7a6dfe5f428b2b593d38585d4/pkg/tsdb/influxdb/flux/executor.go#L86

Reply:
rather than an error I mean something like:
```
frame.AppendNotices(data.Notice{
		Severity: data.NoticeSeverityWarning,
		Text: "Results are truncated based on maxDataPoints",
	})
```